gcp/modules/gcp-stackdriver-monitoring/alert_couchdb_membership_errors.tf

```diff
@@ -7,7 +7,7 @@ resource "google_monitoring_alert_policy" "couchdb_membership_errors" {
       filter          = "metric.type=\"logging.googleapis.com/user/couchdb_membership.error\" resource.type=\"k8s_container\""
       comparison      = "COMPARISON_GT"
       threshold_value = 1.0
-      duration        = "0s"
+      duration        = "3600s"
```

Comment:
1 hour duration?

Reply:
![image](https://user-images.githubusercontent.com/158641/70812551-220a8180-1dc8-11ea-8fce-96fb2478115e.png)

Yes, 30 minutes are not enough for a node upgrade, so the next step is 1 hour.